mapo/agents/mapo/mapo.py

```diff
@@ -10,11 +10,20 @@
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
+
 DEFAULT_CONFIG = with_common_config(
     {
         # === MAPO ===
         # How many samples to draw from the dynamics model
         "branching_factor": 1,
+        # Whether to use the env's dynamics to calculate the actor loss
+        "use_true_dynamics": False,
+        # Which model-learning optimization to use
+        # Valid values: "mle" (Maximum Likelihood Estimation),
+        # "pga" (DPG-aware loss function),
+        "model_loss": "mle",
+        # Which kernel metric to compare gradients in dynamics loss
+        "kernel": "l2",
```

Comment:
As a default kernel I'm currently using "l2", but we can change this anytime...

Reply:
No problem!